idd/Energy+.idd.in

```diff
@@ -68354,6 +68382,8 @@ SolarCollector:FlatPlate:Water,
        \type alpha
        \reference-class-name validBranchEquipmentTypes
        \reference validBranchEquipmentNames
+       \reference-class-name validPlantEquipmentTypes
+       \reference validPlantEquipmentNames
```

Comment:
Not sure solar collectors belong on PlantEquipmentLists? @mitchute @Myoldmopar ?

Reply:
Looks to me like it *should* be there: https://github.com/NREL/EnergyPlus/blob/develop/src/EnergyPlus/SolarCollectors.hh#L128

Reply:
Lots of confusion on what the placement of solar collectors should be, especially since OpenStudio forces you to put the Flat Plate ones on the supply side (for some reason... and it appears to work fine too...)
I agree that generally speaking the documentation and example files in E+ seem to imply demand side placement (with control of the loop via availability managers).

References:

* https://unmethours.com/question/27837/which-side-of-the-plant-loop-for-solar-collector-in-eopenstudio/
* https://github.com/NREL/OpenStudio/issues/2880
* https://github.com/NREL/OpenStudio/issues/3761#issuecomment-553814770

Reply:
Opening a dedicated issue for this instead: https://github.com/NREL/EnergyPlus/issues/7735